src/ViewInterface.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Starlit App.
+ *
+ * @copyright Copyright (c) 2016 Starweb AB
+ * @license   BSD 3-Clause
+ */
+
+namespace Starlit\App;
+
+use Symfony\Component\HttpFoundation\Request;
+use Starlit\App\ViewHelper\AbstractViewHelper;
+
+/**
+ * Abstract implementation of a View
+ */
+interface ViewInterface
+{
+    /**
```

Comment:
Some of the PHPDocs here don't really clarify anything. Can be removed for cleaner code :)